.babelrc

```diff
@@ -10,7 +10,15 @@
       }
     ],
     "@babel/preset-react",
-    "@babel/preset-flow",
-    "@babel/preset-stage-1"
```

Comment:
`@babel/preset-stage-1` was deprecated and removed

https://babeljs.io/blog/2018/07/27/removing-babels-stage-presets